src/friendly-iframe-embed.js

```diff
@@ -580,6 +618,40 @@ export class FriendlyIframeEmbed {
   }
 }
 
+/**
+ * @param {!Window} win
+ * @param {!Element} ampAdParent
+ * @param {!Element} ampLightbox
+ */
+function renderCloseButtonHeader(win, ampAdParent, ampLightbox) {
+  const el = htmlFor(ampAdParent)`
+    <i-amphtml-ad-close-header role=button>
+      <div>Ad</div>
```

Comment:
we need to have some way of personalizing this, maybe a `data-ad-lightbox-text` attr on A4A-only `amp-lightbox`?

Reply:
Is there a common idiom (e.g. for `amp-ad`) to do this?